unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.LoadingScreen
+{
+    /// <summary>
+    ///     Display the current state of the Loading Screen Controller
+    /// </summary>
+    public class LoadingScreenView : BaseComponentView, ILoadingScreenView
+    {
+        private static readonly string PATH = "_LoadingScreen";
+
+        [SerializeField] private LoadingScreenTipsView tipsView;
+        [SerializeField] private LoadingScreenPercentageView percentageView;
+        [SerializeField] private RawImage rawImage;
+        private RenderTexture renderTexture;
+
+        public event Action<ShowHideAnimator> OnFadeInFinish;
+
+        public static LoadingScreenView Create() =>
+            Create<LoadingScreenView>(PATH);
+
+        public override void Start()
+        {
+            base.Start();
```

Comment:
No need of calling the base start method that should be empty in the BaseComponentView